gradio/blocks.py

```diff
@@ -1303,7 +1301,6 @@ def iterate_over_children(children_list):
                     original_mapping[o] for o in dependency["outputs"]
                 ]
                 dependency.pop("status_tracker", None)
-                dependency.pop("zerogpu", None)
```

Comment:
we should leave this as we are loading old configs which may include the zerogpu key int he config